devices/src/virtio/vhost/mod.rs

```diff
@@ -55,3 +55,6 @@ pub enum Error {
 type Result<T> = std::result::Result<T, Error>;
 const INTERRUPT_STATUS_USED_RING: u32 = 0x1;
 const TYPE_VSOCK: u32 = 19;
+
+// VHOST_IRQ_AVAILABLE and KILL_EVENT. KILL_EVENT is unused yet.
+pub const VHOST_EVENTS_COUNT: usize = 2;
```

Comment:
I think it's better to keep this defined next to `VHOST_IRQ_AVAILABLE` and `KILL_EVENT` like it's done in `net` and `block`. The idea is to have them next to each other so that updating/adding any of them doesn't take them out of sync (I mean it's much easier to spot when changing one that you might need to change the other as well).